codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: false
+github_checks:
+  annotations: false
+
+fixes:
+  - "vendor/github.com/docker/dockerfile::" # move path to its original location
```

Comment:
necessary with go submodule otherwise codecov loses track of the module path and looks in vendor dir instead:

![image](https://user-images.githubusercontent.com/1951866/149816290-02176951-1ee4-4388-9d0a-29d5848fbba3.png)